privly.safariextension/Info.plist

```diff
@@ -30,6 +33,8 @@
 	</dict>
 	<key>Description</key>
 	<string>The official Privly Safari browser extension.</string>
+	<key>DeveloperIdentifier</key>
+	<string>34B3M269R6</string>
```

Comment:
This identifier is used for signing the extension so that it can be used as a package on safari